Source/WebKit/WPEPlatform/wpe/WPEApplicationAccessibleAtk.h

```diff
@@ -29,6 +29,7 @@
 #include "WPEToplevel.h"
 #include <atk/atk.h>
 #include <glib-object.h>
+#include <wpe/atk/ATKCompat.h>
```

Comment:
This is a private header, you could use `"AtkCompat.h"`

Reply:
Good point!